src/Connector.sol

```diff
@@ -124,15 +135,33 @@ contract CentrifugeConnector is Test {
         memberlist.updateMember(user, validUntil);
     }
 
-    function transferTo(
+    function transfer(
```

Comment:
Rename to `transferIn`

Reply:
I would strongly suggest going with `transfer` for the public method, the internal method can be anything (something like `handleTransfer` / `onTransfer` / `executeTransfer`?). Reason being: the internal method won't be called by anyone externally, it's just called by the router. So only the name of the external public method really matters, and in this case the intuitive name imo is just `transfer`.

Reply:
Good point! Agreed 👍